packages/firebase_admob/test/firebase_admob_test.dart

```diff
@@ -151,4 +153,53 @@ void main() {
       ]);
     });
   });
+
+  group('$AdInstanceManager', () {
+    final List<MethodCall> methodCallLog = <MethodCall>[];
+
+    setUp(() {
+      adInstanceManager = AdInstanceManager(MethodChannel('test_channel'));
+      adInstanceManager.channel.setMockMethodCallHandler(
+        (MethodCall methodCall) async {
+          methodCallLog.add(methodCall);
+          return null;
+        },
+      );
+    });
+
+    tearDown(() {
+      methodCallLog.clear();
+    });
+
+    test('initialize', () {
+      FirebaseAdMob.instance.initialize(appId: 'testAppId');
+      expect(methodCallLog, <Matcher>[
+        isMethodCall(
+          'initialize',
+          arguments: null,
```

Comment:
`appId` isn't included in here since this is no longer required on Android. On iOS, the parameter is already ignored.

Android tutorial: https://developers.google.com/admob/android/quick-start